packages/shared/auth0-client/src/index.ts

```diff
@@ -31,6 +31,25 @@ export default class Auth0Client {
     this.clientSecret = clientSecret;
   }
 
+  async deleteUser(userId: number): Promise<APIResponse<boolean>> {
```

Comment:
I have a concern over changes in structure between different functions. Ignoring the change in style, this function returns `boolean` to indicate success. By contrast, other functions with empty return types simply return `{}`, since it's the `ResponseStatus` which indicates success or failure to the caller.

In other words, the `boolean` returned by this is never used.